### PR TITLE
MODSOURMAN-384: Implement writing entities processing information to the log

### PR DESCRIPTION
**Pull request by Aliaksandr-Fedasiuk**
## Purpose
Implement writing entities processing information to the log

## Approach
The possibilities of the DataImportJournalConsumersVerticle class have been extended to handle INVENTORY, COMPLETED, and ERROR events.

## Learning
[MODSOURMAN-384](https://issues.folio.org/browse/MODSOURMAN-384)

### CODE REVIEW

File: mod-source-record-manager-server/src/main/java/org/folio/services/RecordProcessedEventHandlingServiceImpl.java [hunk 1]
```diff
@@ -72,7 +72,7 @@ public RecordProcessedEventHandlingServiceImpl(@Autowired JobExecutionProgressSe
     try {
       DataImportEventTypes eventType = DataImportEventTypes.valueOf(dataImportEventPayload.getEventType());
       jobExecutionProgressService.updateJobExecutionProgress(jobExecutionId, progress -> changeProgressAccordingToEventType(progress, eventType), params.getTenantId())
-        .onSuccess(ar -> saveJournalRecordIfNecessary(dataImportEventPayload))
+        //.onSuccess(ar -> saveJournalRecordIfNecessary(dataImportEventPayload))
```
Comment (VRohach) [inline mod-source-record-manager-server/src/main/java/org/folio/services/RecordProcessedEventHandlingServiceImpl.java:75]: Remove this commented line

File: mod-source-record-manager-server/src/main/java/org/folio/verticle/DataImportJournalConsumersVerticle.java [hunk 2]
```diff
@@ -4,21 +4,48 @@
...
   @Autowired
   @Qualifier("DataImportJournalKafkaHandler")
   private AsyncRecordHandler<String, String> dataImportJournalKafkaHandler;
 
-  //TODO MODSOURMAN-384
   @Override
   public List<String> getEvents() {
-    return Collections.singletonList(DI_INVENTORY_INSTANCE_CREATED.value());
+    return List.of(
+      //DI_SRS_MARC_BIB_RECORD_CREATED.value()
```
Comment (VRohach) [inline mod-source-record-manager-server/src/main/java/org/folio/verticle/DataImportJournalConsumersVerticle.java:33]: Remove or uncomment

**Review by VRohach — APPROVED** [review]

File: mod-source-record-manager-server/src/main/java/org/folio/services/RecordProcessedEventHandlingServiceImpl.java (hunk 1 above)
Comment (Aliaksandr-Fedasiuk) [reply mod-source-record-manager-server/src/main/java/org/folio/services/RecordProcessedEventHandlingServiceImpl.java]: Fixed.

File: mod-source-record-manager-server/src/main/java/org/folio/verticle/DataImportJournalConsumersVerticle.java (hunk 2 above)
Comment (Aliaksandr-Fedasiuk) [reply mod-source-record-manager-server/src/main/java/org/folio/verticle/DataImportJournalConsumersVerticle.java]: Fixed.

File: NEWS.md [hunk 3]
```diff
@@ -1,3 +1,6 @@
+## xxxx-xx-xx
+* [MODSOURMAN-384](https://issues.folio.org/browse/MODSOURMAN-384) Implement writing entities processing information to the log
```
Comment (KaterynaSenchenko) [inline NEWS.md:2]: shouldn't it be included in the R1 release (v3.0.0)?

**Review by KaterynaSenchenko — APPROVED** [review]

**Review by RuslanLavrov — APPROVED** [review]

File: mod-source-record-manager-server/src/main/java/org/folio/verticle/consumers/util/JournalParams.java [hunk 4]
```diff
@@ -0,0 +1,158 @@
...
+
+  public JournalParams(JournalRecord.ActionType journalActionType,
+                       JournalRecord.EntityType journalEntityType,
+                       JournalRecord.ActionStatus journalActionStatus) {
+    this.journalActionType = journalActionType;
+    this.journalEntityType = journalEntityType;
+    this.journalActionStatus = journalActionStatus;
+  }
+
+  private interface IJournalParams<H> {
```
Comment (RuslanLavrov) [inline mod-source-record-manager-server/src/main/java/org/folio/verticle/consumers/util/JournalParams.java:26]: could you pls remove the unused type parameter

File: NEWS.md (hunk 3 above)
Comment (Aliaksandr-Fedasiuk) [reply NEWS.md]: Fixed.

File: mod-source-record-manager-server/src/main/java/org/folio/verticle/consumers/util/JournalParams.java (hunk 4 above)
Comment (Aliaksandr-Fedasiuk) [reply mod-source-record-manager-server/src/main/java/org/folio/verticle/consumers/util/JournalParams.java]: Fixed.

**Pull request merged**